participants/rashilakhadka/script.js

```diff
@@ -0,0 +1,70 @@
+document.addEventListener('DOMContentLoaded', () => {
+    const loginForm = document.getElementById('loginForm');
+    const saveCredentialsBtn = document.getElementById('saveCredentials');
+    const logoutBtn = document.getElementById('logout');
+    const feedback = document.getElementById('feedback');
+
+    //check for credential support in browser
+    if(!navigator.credentials || !navigator.credentials.preventSilentAccess) {
+        feedback.textContent = 'Credential Management API is not supported in this browser';
+        saveCredentialsBtn.style.display = 'none';
+        logoutBtn.style.display = 'none';
+        return;
+    }
```

Comment:
The early exit in case if the credential support is not available is a very good practice. 🚀